docs/docs/cmd/purview/retentionlabel/retentionlabel-add.md

```diff
@@ -0,0 +1,128 @@
+```
+
+## Remarks
+
+!!! attention
+    This command is based on a Microsoft Graph API that is currently in preview and is subject to change once the API reached general availability.
+
+## More information
+
+- Create retentionLabel: [https://learn.microsoft.com/en-us/graph/api/security-retentionlabel-post?view=graph-rest-beta&tabs=http](https://learn.microsoft.com/en-us/graph/api/security-retentionlabel-post?view=graph-rest-beta&tabs=http)
+
+## Response
+
+=== "JSON"
+
+    ```json
+    {
+      "displayName": "some label",
+      "descriptionForAdmins": "Description for admins",
+      "descriptionForUsers": "Description for users",
+      "isInUse": false,
+      "retentionTrigger": "dateLabeled",
+      "behaviorDuringRetentionPeriod": "retain",
+      "actionAfterRetentionPeriod": "delete",
+      "createdDateTime": "2022-12-21T09:28:37Z",
+      "lastModifiedDateTime": "2022-12-21T09:28:37Z",
+      "labelToBeApplied": "another label",
+      "defaultRecordBehavior": "startLocked",
+      "id": "f7e05955-210b-4a8e-a5de-3c64cfa6d9be",
+      "retentionDuration": {
+        "days": 365
+      },
+      "createdBy": {
+        "user": {
+          "id": null,
+          "displayName": "John Doe"
+        }
+      },
+      "lastModifiedBy": {
+        "user": {
+          "id": null,
+          "displayName": "John Doe"
+        }
+      },
+      "dispositionReviewStages": []
+    }
+    ```
+
+=== "Text"
+
+    ```text
+    actionAfterRetentionPeriod   : delete
+    behaviorDuringRetentionPeriod: retain
+    createdBy                    : {"user":{"id":null,"displayName":"John Doe"}}
+    createdDateTime              : 2022-12-21T09:32:38Z
+    defaultRecordBehavior        : startLocked
+    descriptionForAdmins         : Description for admins
+    descriptionForUsers          : Description for users
+    displayName                  : some label
+    dispositionReviewStages      : []
+    id                           : 3b388f3c-541b-4696-ad0e-83f960695d89
+    isInUse                      : false
+    labelToBeApplied             : another label
+    lastModifiedBy               : {"user":{"id":null,"displayName":"John Doe"}}
+    lastModifiedDateTime         : 2022-12-21T09:32:38Z
+    retentionDuration            : {"days":365}
+    retentionTrigger             : dateLabeled
+    ```
+
+=== "CSV"
+
+    ```csv
+    displayName,descriptionForAdmins,descriptionForUsers,isInUse,retentionTrigger,behaviorDuringRetentionPeriod,actionAfterRetentionPeriod,createdDateTime,lastModifiedDateTime,labelToBeApplied,defaultRecordBehavior,id,retentionDuration,createdBy,lastModifiedBy,dispositionReviewStages
+    some label,Description for admins,Description for users,,dateLabeled,retain,delete,2022-12-21T09:33:32Z,2022-12-21T09:33:32Z,another label,startLocked,cfc8b132-7aef-45f4-9fcf-3c199090ba2a,"{""days"":365}","{""user"":{""id"":null,""displayName"":""John Doe""}}","{""user"":{""id"":null,""displayName"":""John Doe""}}",[]
```

Comment:
lets remove the empty line

```suggestion
    ```csv
displayName,descriptionForAdmins,descriptionForUsers,isInUse,retentionTrigger,behaviorDuringRetentionPeriod,actionAfterRetentionPeriod,createdDateTime,lastModifiedDateTime,labelToBeApplied,defaultRecordBehavior,id,retentionDuration,createdBy,lastModifiedBy,dispositionReviewStages
    some label,Description for admins,Description for users,,dateLabeled,retain,delete,2022-12-21T09:33:32Z,2022-12-21T09:33:32Z,another label,startLocked,cfc8b132-7aef-45f4-9fcf-3c199090ba2a,"{""days"":365}","{""user"":{""id"":null,""displayName"":""John Doe""}}","{""user"":{""id"":null,""displayName"":""John Doe""}}",[]
```

Reply:
Hi @Adam-it 
The empty line is being shown in the browser. However if you open the file in VS Code, we do not have empty line.
This could be because of very long output on one line.

However, please suggest as appropriate.

Reply:
yes your right 👍. My mistake. sorry for that 🙏